File: src/main/java/com/jinkyumpark/bookitout/search/apiResponse/mille/ApiMillieResponse.java
package com.jinkyumpark.bookitout.search.apiResponse.mille;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor @AllArgsConstructor
@Getter
public class ApiMillieResponse {
    @JsonProperty("RESP_CD")
    private Integer respCd;

    @JsonProperty("RESP_MSG")
    private String respMsg;

    @JsonProperty("RESP_DATA")
    private ApiMillieData respData;

    @JsonProperty("RESP_HOST")
    private String respHost;
}

File: search/src/main/kotlin/com/jinkyumpark/search/subscription/apiResponse/millie/ApiMillieResponse.kt
package com.jinkyumpark.search.subscription.apiResponse.millie

import com.fasterxml.jackson.annotation.JsonProperty

data class ApiMillieResponse(
    @JsonProperty("RESP_CD")
    val respCd: Int?,

    @JsonProperty("RESP_MSG")
    val respMsg: String?,

    @JsonProperty("RESP_DATA")
    val respData: ApiMillieData?,

    @JsonProperty("RESP_HOST")
    val respHost: String?,
)